prototype/storage_api.py

```diff
@@ -0,0 +1,175 @@
+from mmif import Mmif
+from clams import mmif_utils
+from flask import Flask, request, jsonify, send_from_directory
+from enum import Enum
+from pydantic import BaseModel
+from typing import List, Dict
+from typing_extensions import Annotated
+import os
+import yaml
+import hashlib
+import json
+
+
+app = Flask(__name__)
+# get post request from user
+# read mmif inside post request, get view metadata
+# store in nested directory relating to view metadata
+
+# TODO: this app accepts "unresolvable" as an app version number; it needs to be fixed because
+# TODO: "unresolvable" is not specific and can represent multiple versions.
+
+
+@app.route("/")
+def root():
+    return {"message": "Storage api for pipelined mmif files"}
+
+
+@app.route("/upload_mmif/", methods=["POST"])
```

Comment:
let's think about the routings here. Existing "baapb resolver" codebase (`api/__init__.py`) uses many routings, but only one of them (`/searchapi/`) is a pure API landing point (others are attached to some web page with minimal front end implementation). 

Eventually, we want to run a single server app that can handle "assets" (video files) as well as can handle "mmif", meaning this `storage_api.py` should be merged into the `api/` pacakge. To that end, I think we need some basic rules for naming these routing paths. 

Since `/searchapi` is already used in production server (for `baapb://` resolution), how about 
- `/upload_mmif` >> `/storeapi/mmif`
- `/retrieve` >> `/searchapi/mmif`
?

Note that changes in the routing names will also impact the client-side implementation (that you're currently working on).